util-core/src/main/scala/com/twitter/util/BatchExecutor.scala

```diff
@@ -22,9 +22,8 @@ import scala.collection.mutable
  * - Rather than having separate sizeThreshold and sizePercentile parameters,
  *   just have a single call-by-name sizeThreshold parameter, and let the
  *   caller implement whatever logic they want to compute the next batch size.
- * - Return an instance of a new class (Batcher?) which extends
- *   `In => Future[Out]`. Could support things like querying the queue size,
- *   forcing a flush, attaching callbacks to flush operations, etc.
+ * - Add more functionality to class Batcher. Could support things like querying the queue size,
```

Comment:
The only TODOs I could find in Future.scala were about future deprecation, which is actually probably of interest to the end-user.  My general philosophy is that TODOs that are about changing semantics are for librarians, so they should be in normal comments.

That formatting is fine, I think just sticking it in the line beneath the constructor for BatchExecutor is good.

EDIT: actually, I just remembered that BatchExecutor is private, so it won't get published as scaladoc anyway.  I don't feel as strongly now, so feel free to ignore this.